CactusCareApi/CactusCare.BLL/BLLModule.cs

```diff
@@ -41,6 +37,13 @@ protected override void Load(ContainerBuilder builder)
 
             builder.RegisterType<JwtTokenGenerator>()
                 .SingleInstance();
+
+            //configure validation
+            builder.RegisterType<FluentValidationService>().As<IValidationService>();
+            builder.RegisterType<AutofacValidatorFactory>().As<IValidatorFactory>();
+            builder.RegisterAssemblyTypes(Assembly.GetExecutingAssembly())
```

Comment:
add .SingleInstance here or validators, validationservice and ValidatorFactor will be created for each request